libs/common/src/ui-scripts/disable-form-submit-on-submission.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Gets the last form on the page if any forms are present.
+ * The last form is taken as it is foreseeable to have multiple forms on a page,
+ * and the submit button should always be in the last form on a page.
+ * @returns {HTMLFormElement | null}
+ */
+const getLastFormIfPresent = () => {
+  const forms = document.querySelectorAll('form');
+
+  if (!forms.length) {
+    return null;
+  }
+
+  const lastForm = forms[forms.length - 1];
+
+  if (!lastForm) {
+    return null;
+  }
+
+  return lastForm;
+};
+
+const validateAndDeleteElement = (elementToDelete) => {
+  if (elementToDelete instanceof HTMLElement) {
+    elementToDelete.remove();
+  }
+};
+
+const validateAndDisableSubmitter = (submitter) => {
+  if (submitter instanceof HTMLElement) {
+    submitter.setAttribute('disabled', '');
+    submitter.setAttribute('aria-disabled', 'true');
+  }
+};
+
+const validateAndEnableSubmitter = (submitter) => {
+  if (submitter instanceof HTMLElement) {
+    submitter.removeAttribute('disabled');
+    submitter.removeAttribute('aria-disabled');
+  }
+};
+
+/**
+ * Creates a hidden input of the submitter used to submit the form.
+ * This hidden input copies any name or value attributes that the submission submitter has,
+ * as when we disable the initial submitter, the disabled submitter will not submit
+ * this information to the server.
+ * @param {HTMLElement} submitter
+ */
+const createHiddenInputOfSubmitter = (submitter) => {
+  const hiddenInput = document.createElement('input');
+
+  hiddenInput.setAttribute('id', 'resubmit-prevention-hidden-input');
+  hiddenInput.setAttribute('type', 'hidden');
```

Comment:
Think this is ok accessibility-wise but just want to double check we dont need `aria-hidden` or similar

Reply:
`hidden` should cover this